File: official/docs/python/v7/referral-customers/add-payment-method-with-bank-account.py
import os

import easypost


easypost.api_key = os.getenv("EASYPOST_API_KEY")

payment_method = easypost.beta.Referral.add_payment_method(
    stripe_customer_id="cus_...",
    payment_method_reference="ba_...",
    primary_or_secondary="primary",
)

print(payment_method)

File: official/docs/node/v7/referral-customers/add-payment-method-with-bank-account.js
const EasyPostClient = require('@easypost/api');

const client = new EasyPostClient('EASYPOST_API_KEY');

(async () => {
  const paymentMethod = await client.BetaReferralCustomer.addPaymentMethod('cus_...', 'ba_...');

  console.log(paymentMethod);
})();
